src/velero/README.md

```diff
@@ -129,6 +129,5 @@ velero restore create uds-restore-$(date +%s) \
 > before running the restore.
 
```

Comment:
This is more a note for @defenseunicorns/uds-core - we will probably want to move the contents of this readme itself onto the docs site in the future.